ndb/tests/unit/test_async_.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from google.cloud.ndb import _eventloop
+from google.cloud.ndb import async_
+
+
+@unittest.mock.patch("google.cloud.ndb.async_._eventloop.EventLoop")
+def test_async_context(EventLoop):
+    one = unittest.mock.Mock()
+    two = unittest.mock.Mock()
+    EventLoop.return_value = one
```

Comment:
You could do `EventLoop.side_effect = [one, two]` rather than switching the `return_value` down below

Reply:
I knew that, but then didn't know that for some reason.